README.md

```diff
@@ -14,11 +14,11 @@ The main modules that this crate exposes are:
 Currently v1.8.x nightly is required
 ```bash
 $ rustc -V
-rustc 1.8.0-nightly (fae516277 2016-02-13)
+rustc 1.8.0-nightly (c92e910c1 2016-02-21)
```

Comment:
When you ask multirust to install a specific version like: multirust override nightly-2016-02-22 could end up in that specific rustc version.

I mean, after doing the multirust with the 02-22 version, the rustc version is that one. At least on mac.